app/app.js

```diff
@@ -121,6 +121,10 @@ window.run = function() {
             HeaderConfig.cartHeaderContent.imageUrl,
             HeaderConfig.cartHeaderContent.id
         ).then(function(counterBadgeController) {
+            Astro.registerRpcMethod(AstroRpc.names.updateCartCounter, ['count'], function(res, count) {
```

Comment:
Talked to @jeremywiebe and we discussed renaming `AstroRpc` to `AppRpc` in the scaffold and adding an `AstroRpc` in astro.